tests/test_runners/test_minitest.py

```diff
@@ -19,3 +20,23 @@ def test_record_test_minitest(self):
 
         expected = self.load_json_from_file(self.result_file_path)
         self.assert_json_orderless_equal(expected, payload)
+
+    @responses.activate
+    def test_record_test_minitest_chunked(self):
+        # override max chunk
+        module = sys.modules['launchable.commands.record.tests']
+        module.CASE_POST_CHUNK = 5
```

Comment:
Don't we need to restore this value at the end of the test so as not to influence other tests?

Alternatively, given that this is only used locally, maybe easier to turn this into a hidden option. That way a test will avoid a side effect, and the option might be marginally useful as a last resort in some desparate situations, too.